doc/gateway-instrumentation.md

```diff
@@ -0,0 +1,133 @@
+## Gateway instrumentation
+
+3scale Operator can enable insturmentation on it's managed APIcasts by using the [OpenTelemetry](https://opentelemetry.io/) SDK.
+More specifically, enabling the [Nginx opentelemetry tracing library](https://github.com/open-telemetry/opentelemetry-cpp-contrib/tree/main/instrumentation/nginx).
+
+It works with Jaeger since version **1.35**.  If the existing collector does not support
+OpenTelemetry traces, an OpenTelemetry Collector is required as tracing proxy.
+
+Supported propagation types: [W3C](https://w3c.github.io/trace-context/)
+
+### Prerequisites
+
+* Opentelemetry Collector supporting the APIcast exporter. Currently, the only implemeneted [exporter](https://opentelemetry.io/docs/reference/specification/protocol/exporter/)
+in APIcast is OTLP over gRPC `OTLP/gRPC`. Even though OpenTelemetry specification supports also OTLP over HTTP (`OTLP/HTTP`),
+this exporter is not included in APIcast. It works with Jaeger since version **1.35**.
+
+For dev/testing purposes, you can deploy quick&easy Jaeger with few commands. **Not suitable** for production use, though.
+Create and connect with the cluster.
+
+```
+❯ make jaeger-deploy
+```
+
+That should deploy Jaeger service listening at `jaeger:4317`.
+
+### Create secret with the APIcast instrumentation configuration
+
+The configuration file specification is defined in the [Nginx instrumentation library repo](https://github.com/open-telemetry/opentelemetry-cpp-contrib/tree/main/instrumentation/nginx).
+
+`otlp` is the only supported exporter.
+
+The host/port address is set according to the dev/testing deployment defined in this repo.
+Change it to whatever you have your collector deployed.
+
+The name of the secret. `otel-config` in the example, will be used in the APIcast CR.
+
+```yaml
+kubectl apply -f - <<EOF
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: otel-config
+type: Opaque
+stringData:
+  config.json: |
```

Comment:
Made a comment about this in the verification steps about the content not being json.
I guess it's just a little confusing 
Easier to respond here

Reply:
suspect if it was just 
```yaml
stringData:
  config.toml: | 
```
it would work as well

Reply:
I've updated docs, thanks Austin

Reply:
yep confirmed with Michal that changing it to toml works.